spotify-conversion-next/pages/api/auth/[...nextauth].ts

```diff
@@ -0,0 +1,11 @@
+import NextAuth from "next-auth";
+import SpotifyProvider from "next-auth/providers/spotify";
+
+export default NextAuth({

```

Comment:
Countless hours wasted trying to manually do what this shit is just doing by default in 10 lines of code. Frick me